convert_lora_to_gguf.py

```diff
@@ -277,10 +277,15 @@ def parse_args() -> argparse.Namespace:
     return parser.parse_args()
 
 
-def load_hparams_from_hf(hf_model_id: str) -> dict[str, Any]:
+def load_hparams_from_hf(hf_model_id: str) -> tuple[dict[str, Any], Path | None]:
+    from huggingface_hub import try_to_load_from_cache
+
     # normally, adapter does not come with base model config, we need to load it from AutoConfig
     config = AutoConfig.from_pretrained(hf_model_id)
-    return config.to_dict()
+    cache_dir = try_to_load_from_cache(hf_model_id, "config.json")
+    cache_dir = Path(cache_dir).parent if cache_dir is not None else None
```

Comment:
```suggestion
    cache_dir = Path(cache_dir).parent if isinstance(cache_dir, str) else None
```
The reason we test is because the value may be `_CACHED_NO_EXIST `, see [docs](https://huggingface.co/docs/huggingface_hub/v1.1.4/en/package_reference/cache#huggingface_hub.try_to_load_from_cache).